lib/ansible/template/native_helpers.py

```diff
@@ -95,12 +157,20 @@ def ansible_native_concat(nodes):
     if not head:
         return None
 
+    unsafe = False
+
     if len(head) == 1:
         out = head[0]
 
+        unsafe = _is_unsafe(out)
+
         # TODO send unvaulted data to literal_eval?
         if isinstance(out, AnsibleVaultEncryptedUnicode):
-            return out.data
+            out = out.data
+            if unsafe:
+                out = wrap_var(out)
```

Comment:
Same deal here- that wrapper type [has `__UNSAFE__` hardcoded true on it already](https://github.com/ansible/ansible/blob/devel/lib/ansible/parsing/yaml/objects.py#L73) (for no good reason IMO, at least in DT, we've killed that), so should be able to just unconditionally return it in this case.